azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participant/grid/screenshare/GestureListener.kt

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.presentation.fragment.calling.participant.grid.screenshare
+
+import android.content.Context
+import android.graphics.PointF
+import android.view.GestureDetector
+import android.view.MotionEvent
+import android.view.ScaleGestureDetector
+
+/**
+ * The GestureListener detects click as well as scale and notify zoom frame layout to apply transformation
+ * Used some parts of https://github.com/facebook/fresco zoomable to implement pinch and zoom
```

Comment:
@ahammer @pavelprystinka I indicated help here, please let me know if it looks good.

Reply:
lgtm